HTML/2. headings.md

```diff
@@ -0,0 +1,19 @@
+# Headings and Paragraphs
+
+There are 6 types of headings in HTML - `h1` through `h6`
+
+`h1` being the biggest and `h6` being the smallest
```

Comment:
These shouldn't be categorized by size, but by semantics and meaning, considering that CSS can completely invert the sizes, make them all equal, etc.

Reply:
Oh, this is mentioned afterwards, how about my mixing them, or deferring focus on the sizes?